fedireads/connectors/abstract_connector.py

```diff
@@ -266,25 +267,25 @@ def update_from_mappings(obj, data, mappings):
         if key == 'id':
             continue
 
-        if has_attr(obj, key):
+        try:
+            hasattr(obj, key)
```

Comment:
Is this code right? I don't expect hasattr to raise ```ValueError```

Is this not just:

```
if not hasattr(obj, key):
   setattr(obj, key, formatter(value))
```

Reply:
no, you're right, it doesn't make sense. I've fixed it subsequently (and I also switched update form mappings to be a whitelist of fields, btw)